imperfect-perfect.tex

```diff
@@ -702,6 +700,21 @@ \section{Cohomology of $(\varphi, \Gamma)$-modules}
 (By contrast, the maps $\tilde{\calR}^{[s,r]}_{\QQ_p,A} \to \tilde{\calR}^{[s',r']}_{\QQ_p,A}$ are not completely continuous.)
 \end{remark}
 
+\begin{defn}
+Let $\calR_{K, A}(1)$ denote the object of $\PhiGamma_{K, A}$ corresponding to the cyclotomic character in $\Rep_A(G_K)$.
+There is then a canonical residue map
+\[
+\Res: H^2_{\varphi, \Gamma}(\calR_{K,A}(1)) \to A
+\]
+which when $[K: \QQ_p] < \infty$ corresponds to the residue map from Tate local duality.
```

Comment:
Do we use the residue map?

Also, I'm not sure this map exists when K/Qp is infinite.  Say A = Qp and K = F.  When F/Qp is finite, I think one takes the coefficient of pi^-1 and applies Tr_{F/Qp}, perhaps divided by [F:Qp].  When F/Qp is infinite, how does one replace the trace?